test/integration/config.go

```diff
@@ -29,8 +29,8 @@ type config struct {
 	MappingTableID         string        `env:"INTEG_TEST_MAPPING_TABLE_ID,required"`
 	MappingFailureTableID  string        `env:"INTEG_TEST_MAPPING_FAILURE_TABLE_ID,required"`
 	PolicyTableID          string        `env:"INTEG_TEST_POLICY_TABLE_ID,required"`
-	QueryRetryWaitDuration time.Duration `env:"INTEG_TEST_QUERY_RETRY_WAIT_DURATION,default=5s"`
-	QueryRetryLimit        uint64        `env:"INTEG_TEST_QUERY_RETRY_COUNT,default=5"`
+	QueryRetryWaitDuration time.Duration `env:"INTEG_TEST_QUERY_RETRY_WAIT_DURATION,default=10s"`
+	QueryRetryLimit        uint64        `env:"INTEG_TEST_QUERY_RETRY_COUNT,default=30"`
```

Comment:
I'm still concerned about what's causing this big difference. Let's first see if the test passes

Reply:
The tests are passed now.